src/utils/constants.ts

```diff
@@ -150,11 +150,15 @@ export const VAULT: Record<number, Address> = {
 };
 
 export const VAULT_V3: Record<number, Address> = {
-    [ChainId.SEPOLIA]: '0xBC582d2628FcD404254a1e12CB714967Ce428915',
+    [ChainId.SEPOLIA]: '0xbA1333333333a1BA1108E8412f11850A5C319bA9',
```

Comment:
I don't think we can update SEPOLIA until we have the test pools, etc deployed which may take some time. If we need a release before then provided there are no breaking changes resulting from updated ABIs I think we can keep previous SEPOLIA and add MAINNET/GNOSIS.

Reply:
that makes sense :ballot_box_with_check: